models.go

```diff
@@ -110,3 +112,12 @@ type CommitObject struct {
 type ChangedFileObject struct {
 	Path string
 }
+
+// LabelObject represents the GraphQL label node.
+// https://developer.github.com/v4/object/label
+type LabelObject struct {
```

Comment:
As far as I can tell, the only value we are actually using from the `LabelObject` is `Name`. We should not fetch the other values if that is the case, since it just makes the operation more expensive 🤷‍♂

Reply:
Removed all the others and only left `Name` :+1: